### DISCUSSION TITLE
Second release candidate of Jamulus 3.8.0 has been released

### DISCUSSION THREAD

**Discussion by softins** [jamulussoftware/jamulus#1784] [Announcements]
We have just released version **3.8.0rc2** of Jamulus, and would be happy for you to download, compile and test out this version. This fixes some issues that were present in the rc1 version. If all is well, this will become the 3.8.0 release.

Please note that there are now two different packages for Mac. The standard build `jamulus_3.8.0rc2_mac.dmg`
 should be used on macOS versions from High Sierra (10.13) to Big Sur (11). The legacy build `jamulus_3.8.0rc2_mac_legacy.dmg`
 should only be used on macOS Sierra (10.12), El Capitan (10.11) or Yosemite (10.10).

You can view the Release Notes and download versions for Windows, Mac, Linux (Debian/Ubuntu) and Android (still experimental) at https://github.com/jamulussoftware/jamulus/releases/tag/r3_8_0rc2. You can also download the source code as a zip or tarball for compiling, or you can check out the code using git.

If you find any issues with it, please [raise an Issue](https://github.com/jamulussoftware/jamulus/issues/new/choose) with as many details as possible.

**Comment by jujudusud** [comment]
Hi here and many thanks for the good app Jamulus is !
Just to let you that all seems to be ok here on Archlinux in French except some translations on the go ;-) #1790
Regards.
